ui/pages/AnvilWorkspace/components/LoadWorkspaceDataForm.jsx

```diff
@@ -31,10 +27,10 @@ const BLANK_EXPORT = {
 }
 
 const UploadPedigreeField = React.memo(() =>
-  <div>
-    {/* eslint-disable-next-line jsx-a11y/label-has-for */}
-    <StyledLabel>Upload Pedigree Data</StyledLabel>
-    <Segment>
+  [
```

Comment:
you should probably make this change in the main ui branch as well

Reply:
It is already in the ui branch. Maybe, it is the github's bug that it doesn't show basing on the latest UI branch.

Reply:
you need to pull that branch in to this branch every time you update the other branch. Theres a reason I usually try to get branches merged as quickly as possible, rather than keeping branches off of branches for a long time